src/libraries/System.Private.CoreLib/src/System/Globalization/CalendarData.iOS.cs

```diff
@@ -18,6 +18,15 @@ private bool LoadCalendarDataFromNative(string localeName, CalendarId calendarId
             sNativeName = GetCalendarInfoNative(localeName, calendarId, CalendarDataType.NativeName);
             sMonthDay = GetCalendarInfoNative(localeName, calendarId, CalendarDataType.MonthDay);
             saShortDates = GetCalendarInfoNative(localeName, calendarId, CalendarDataType.ShortDates).Split("||");
+            // Handle ShortDatePattern to have "yyyy" year format
+            List<string> shortDatePatternList = new List<string>(saShortDates);
+            for (int i = 0; i < shortDatePatternList.Count; i++)
+            {
+                shortDatePatternList[i] = NormalizeDatePattern(shortDatePatternList[i]);
```

Comment:
If we skiped `NormalizeDatePattern` then wouldn't it work? Or is it that it `NormalizeDatePattern` handles only `y -> yyyy` and `FixDefaultShortDatePattern` only `yy -> yyyy`?